src/NServiceBus.Core/StaticHeaders/StaticHeadersConfigExtensions.cs

```diff
@@ -0,0 +1,33 @@
+namespace NServiceBus
+{
+    using NServiceBus.StaticHeaders;
+
+    /// <summary>
+    /// Extensions to the public configuration api
+    /// </summary>
+    public static class StaticHeadersConfigExtensions
+    {
+        /// <summary>
+        /// Allows static outgoing headers to be set
+        /// </summary>
+        /// <param name="config">Configuration object</param>
+        /// <param name="key">The header key</param>
+        /// <param name="value">The header value</param>
+        public static void SetOutgoingHeader(this BusConfiguration config, string key, string value)
```

Comment:
+1 for a better name

`AddHeaderToAllOutgoingMessages(string key, string value)` ?